web/client/localConfig.json

```diff
@@ -71,6 +71,48 @@
                 "title": "GeoSolutions GeoServer CSW",
                 "autoload": true
               },
+              "default_map_backgrounds": {
```

Comment:
Where is used? I think it can be removed.

Reply:
Default backgrounds are configured from here

Reply:
Sorry, I was sure that I had removed this comment, but I didn't . Going to resolve this comment